types/redis-rate-limiter/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for redis-rate-limiter 1.0.3
+// Type definitions for redis-rate-limiter 1.2.0
```

Comment:
Two requests:
- please remove the patch version (it should just be 1.2)
- Please remove the line that says `dtheader: false` from the `tslint.json` file in this folder

It would be great to remove all such lines from the `tslint.json`, but that could be a lot of work, and I see it's your first time contributing.